foreshadow/steps/flattener.py

```diff
@@ -75,12 +50,16 @@ def transform(self, X, *args, **kwargs):
             A transformed dataframe.
 
         """
-        # if self._empty_columns is None:
-        #     raise ValueError("Cleaner has not been fitted yet.")
+        return super().transform(X=X)
```

Comment:
Do we still need to override this method?

Reply:
No I'll remove it.